### PR TITLE
Add Path of Exile protocol dissector

### PR DESCRIPTION
**Pull request by 0xA50C1A1**
Please sign (check) the below before submitting the Pull Request:

- [x] I have signed the ntop Contributor License Agreement at https://github.com/ntop/legal/blob/main/individual-contributor-licence-agreement.md
- [x] I have read the contributing guide lines at https://github.com/ntop/nDPI/blob/dev/CONTRIBUTING.md
- [x] I have updated the documentation (in doc/) to reflect the changes made (if applicable)

Describe changes:

Path of Exile is a free-to-play online Action RPG.

### CODE REVIEW

**Comment by 0xA50C1A1** [comment]
again this bug #2258

**Comment by IvanNardi** [comment]
The two issues (fuzzing and windows) are un-related to this PR...

**Comment by 0xA50C1A1** [comment]
> The two issues (fuzzing and windows) are un-related to this PR...

Yeah, I noticed.

File: doc/protocols.rst [hunk 1]
```diff
@@ -656,3 +656,12 @@ References: `RFC <https://datatracker.ietf.org/doc/html/rfc5880>`_
 Traffic of various NetEase games.
 
 References: `Main site <https://www.neteasegames.com/>`_
+
+
+.. _Proto 402:
```
Comment (IvanNardi) [inline doc/protocols.rst]: 403

**Review by IvanNardi — CHANGES_REQUESTED** [review]
Just a minor issue.

File: doc/protocols.rst (hunk 1 above)
Comment (0xA50C1A1) [reply doc/protocols.rst]: Ty, I've fixed it.

**Review by IvanNardi — APPROVED** [review]
Thanks

**Pull request merged**